src/InteractiveIframe/iframeCard.jsx

```diff
@@ -107,6 +108,7 @@ IframeCard.propTypes = {
   eventIds: PropTypes.arrayOf(PropTypes.string),
   url: PropTypes.string.isRequired,
   params: PropTypes.instanceOf(Object),
+  externalProps: PropTypes.instanceOf(Object),
```

Comment:
are params and externalParams instance of object? Not array?

Reply:
They are both objects (it has been this way in Mosaic since the feature was introduced)